src/ess/reflectometry/normalize.py

```diff
@@ -110,9 +112,9 @@ def reflectivity_over_q(
     :
         Reflectivity as a function of Q
     """
-    return NormalizedIofQ(
-        da.bin(Q=qbins, dim=da.dims) / sc.values(n.hist(Q=qbins, dim=n.dims))
-    )
+    reflectivity = da.bin(Q=qbins, dim=da.dims) / sc.values(n.hist(Q=qbins, dim=n.dims))
+    reflectivity.coords['Q_resolution'] = qres.data
```

Comment:
Do you want to add a test that checks that the result has a `Q_resolution` coordinate?
This would avoid this being lost in a future refactor?